HOMEWORK_18/EX1.CPP

```diff
@@ -0,0 +1,155 @@
+#include <iostream>
+#include <memory>
+
+enum class DrinkType
+{
+    Espresso,
+    Latte,
+    Cappuccino,
+    Americano,
+    BlackTea,
+    GreenTea
+};
+
+enum class Size
+{
+    M,
+    L,
+    XL
+};
+
+class Drink
```

Comment:
All of them should be separeted by Class.h/Class.cpp
at least for:
Drink.h/cpp
Coffee.h/cpp
Tea.h/cpp
CoffeeShop.h/cpp
and main.cpp